nextstrain/cli/runner/docker.py

```diff
@@ -387,7 +387,12 @@ def test_image_version():
         ('docker run works',
             test_run()),
         *test_memory_limit(),
-        *test_image_version()
+        *test_image_version(),
+
+        # Rosetta 2 is optional, so convert False (fail) → None (warning)
+        *[(msg, None if status is False else status)
+            for msg, status
+             in test_rosetta_enabled("Rosetta 2 is enabled for faster execution (optional)")],
```

Comment:
At first I wondered, why not return a single `RunnerTestResult` instead of list from `test_rosetta_enabled()`? Then I realized that it allows same-line list comprehension conversion like you've done here. But what about for `*test_memory_limit()` which could be simply `test_memory_limit()`?

```suggestion

```

Reply:
I think `*test_memory_limit()` (and `*test_image_version()`) may be historic holdovers from a previous iteration of the code here where those functions sometimes returned `[(msg, status)]` but other times returned `[]`. I haven't dug into history though…

Reply:
Note that `test_rosetta_enabled()` returns a list of results because it doesn't always want to include a Rosetta test on all platforms:

https://github.com/nextstrain/cli/blob/25075faebad904ff3d9df9422abb16bedb662f80/nextstrain/cli/util.py#L611-L612